operator/src/main/resources/application.properties

```diff
@@ -41,13 +41,20 @@ ingresscontroller.max-ingress-connections=108000
 ingresscontroller.peak-connection-percentage=100
 
 # Enables haproxy option contstats in the kas ingress so that stats are reported live rather than at connection close (RFE-3007 provide a proper mechanism to enable this option).
-ingresscontroller.ingress-container-command=/usr/bin/bash,-c,awk '{print $0} /^defaults$/ {print \"  option contstats\"}' < $TEMPLATE_FILE > /tmp/haproxy-config.template; exec /usr/bin/openshift-router --v=2 --template /tmp/haproxy-config.template
+ingresscontroller.ingress-container-command=/usr/bin/bash,-c,awk '{print $0} /^defaults$/ {print \"  option contstats\"}' < $TEMPLATE_FILE > /tmp/haproxy-config.template; exec /usr/bin/openshift-router --v=2 --template /tmp/haproxy-config.template --commit-interval=${ingresscontroller.commit-interval} --blueprint-route-pool-size=${ingresscontroller.blueprint-route-pool-size} --max-dynamic-servers=${ingresscontroller.max-dynamic-servers}
 # Disconnect established connections after a haproxy reconfiguration (NE-879 should eliminate the need for this)
 ingresscontroller.hard-stop-after=5s
 # Coalesce up-to reloadInterval worth of haproxy updates.  This prevents a flurry of haproxy restarts (1 per OpenShift Route) as each kafka
 # instance is created (NE-879 should eliminate the need for this).
 ingresscontroller.reload-interval-seconds=60
 
+# HA proxy dynamic config manager
+ingresscontroller.dynamic-config-manager=true
+# the following three variables are meaningful when the dynamic-config-manager=true
+ingresscontroller.commit-interval=9223372036854775807ns
+ingresscontroller.blueprint-route-pool-size=250
```

Comment:
Lets pad this - if for some reason we need to add another route per kafka, or support using an entire osd for standard, this won't be sufficient.  How about 500?

Reply:
Done.  I actually tested 500 yesterday. I forgot to push the change.   Memory use is stable at around 150Mi-160Mi per ingress pod.

Reply:
500 makes sense to me